src/block.h

```diff
@@ -24,6 +24,19 @@ class BlockFinder
     //! \a nullptr is returned.
     //!
     CBlockIndex* FindByHeight(int height);
+
+    //!
+    //! \brief Find block by time.
+    //! 
+    //! Traverses the chain in the same way as FindByHeight() and stops at the
+    //! block which is not older than \p time, or the youngest block if it is
+    //! older than \p time.
+    //! 
+    //! \param time Block time to search for.
+    //! \return The youngest block which is not older than \p time, or the
+    //! head of the chain if it is older than \p time.
+    //!
+    CBlockIndex* FindByTime(int64_t time);
```

Comment:
From the name of the function it is unclear whether it returns highest block that is older than requested or it returns lowest block that is more recent than requested.

Reply:
You're right, it's too ambiguous. I will try to figure out a better name.